CHANGELOG

```diff
@@ -1,10 +1,22 @@
+snowplow-web 0.12.3 (2022-12-28)
```

Comment:
```suggestion
snowplow-web 0.12.3 (2023-01-05)
```

Reply:
will set to tomorrow as not going to release tonight, good catch!

Reply:
otherwise good to go!